tests/test_lldp.py

```diff
@@ -97,8 +97,10 @@ def test_subtype_lldp_loc_man_addr_table(self):
 
     def test_subtype_lldp_rem_man_addr_table(self):
```

Comment:
>test_subtype_lldp_rem_man_addr_table [](start = 8, length = 36)

Could you add some more testcases?
1. one ipv4 + one ipv6
2. multiple ipv4
3. multiple ipv6

Reply:
Actually I had some test cases in our private repo. But during upstream, I noticed that this PR i.e. https://github.com/Azure/sonic-snmpagent/pull/112 which removed those sections like testing test_ipv6_rem_man_addr, test_ipv4_rem_man_addr as they are non-accessible nodes. 

kindly suggest how to go ahead with this. I will add the test case if required.

Reply:
Could you test remaining OIDs such as lldpRemManAddrIfSubtype, lldpRemManAddrIfId, lldpRemManAddrOID.

---
In reply to: [444556755](https://github.com/Azure/sonic-snmpagent/pull/136#discussion_r444556755) [](ancestors = 444556755)

Reply:
Added as requested.

Reply:
This comment is still applicable:

Could you add some more testcases?
1. one ipv4 + one ipv6
2. multiple ipv4
3. multiple ipv6

---
In reply to: [445265061](https://github.com/Azure/sonic-snmpagent/pull/136#discussion_r445265061) [](ancestors = 445265061,444556755)

Reply:
Should I add those test cases as they were removed by this PR i.e. #112?

Reply:
That PR makes sense so no need to add back. You can improve based on existing test cases.

---
In reply to: [449309530](https://github.com/Azure/sonic-snmpagent/pull/136#discussion_r449309530) [](ancestors = 449309530,445265061,444556755)